src/formula.cpp

```diff
@@ -4773,6 +4779,11 @@ static std::string debugSubexpressionTypes(ConstFormulaPtr & fml)
 			int consume_backwards = 0;
 			std::string op_name(op->begin,op->end);
 
+			if (op_name == "is" && op > i1 && op + 1 > i1 &&
```

Comment:
You don't need to check op + 1 > i1 since op > i1 already guarantees this, however you *should* check that op + 1 < i2 before using op + 1 to make sure we don't run off the end of the buffer.